modules/jobs/dbInit.tf

```diff
@@ -0,0 +1,115 @@
+resource "kubernetes_job_v1" "db-initialization" {
+  metadata {
+    name      = "db-initialization"
+    namespace = "db"
+  }
+  spec {
+    template {
+      metadata {}
+      spec {
+        container {
+          name    = "swoop-db"
+          image   = "quay.io/element84/swoop-db"
+          command = ["python", "opt/swoop/db/scripts/db-initialization.py"]
+
+          env {
+            name  = "PGHOST"
+            value = "postgres"
+          }
+
+          env {
+            name  = "PGUSER"
+            value = "postgres"
+          }
+
+          env {
+            name  = "PGDATABASE"
+            value = "swoop"
+          }
+
```

Comment:
You might also want to add the `PGPORT`, just to be explicit about that common setting.

It also seems like these connection values likely need to come from the same vars being passed into postgres, so you might see about making them come from inputs and if there's a way to share the values from those postgres inputs (where postgres is part of the deployment) to set them.